### DISCUSSION TITLE
Reached ★100

### DISCUSSION THREAD

**Discussion by tadashi-aikawa** [tadashi-aikawa/obsidian-various-complements-plugin#121] [Announcements]
Today, the number of GitHub stars reached 💯!!

Thanks to our users who use Various Complements and contribute to GitHub Issues!! 🙏 

Since I'm here, I started GitHub Discussions. I have never used GitHub Discussions, but I hope I can learn to use it little by little!

**Comment by FelipeRearden** [comment]
Congratulations @tadashi-aikawa !!!!!!!!!

I just wanna take this event to say thank you very much for everything that you do our community. 

VC and AQS are plugins that changes our Obsidian lives. Your work bring joy to my life everyday 💙 

You deserve ALL ⭐️ ⭐️ ⭐️ from the 🌌

⭐️ ⭐️ ⭐️ ⭐️ ⭐️ ⭐️ ⭐️ ⭐️ ⭐️ ⭐️ ⭐️ ⭐️ ⭐️ ⭐️ ⭐️ ⭐️ ⭐️ ⭐️ ⭐️ ⭐️ ⭐️ ⭐️ ⭐️ ⭐️ ⭐️ ⭐️ ⭐️ ⭐️ ⭐️ ⭐️ ⭐️ ⭐️ ⭐️ ⭐️ ⭐️ ⭐️ ⭐️ ⭐️ ⭐️ ⭐️

I wish you a fantastic day 🌞

**Comment by tadashi-aikawa** [comment]
Thank you, @FelipeRearden!!

I am always grateful for your help and ideas 😄 Have a nice day!! 👍
